client/client.go

```diff
@@ -200,8 +200,8 @@ func (c *Client) CmdSend(status chan *CmdStatus, args ...string) {
 	}
 }
 
-func (c *Client) createTx(receiver common.Address, amount *big.Int) (tee.Transaction, error) {
-	block := atomic.LoadUint64(&c.currentBlock) + 1
+func (c *Client) createTransfer(receiver common.Address, amount *big.Int) (tee.Transaction, error) {
+	block := atomic.LoadUint64(&c.lastBlock) + 1
```

Comment:
why the rename? you create a tx, not a transfer

Reply:
i thought `Transaction` sound more like a general on-chain transaction, whereas this creates a TEE balance transfer.